pkg/sql/logictest/testdata/logic_test/alter_column_type

```diff
@@ -523,3 +523,25 @@ SELECT x FROM t28 ORDER BY x
 5
 10
 15
+
+# Regression 50277, ensure ColumnConversionValidate type conversion doesn't
+# error before running the online schema change.
+statement ok
+CREATE TABLE t29 (x INT8);
+INSERT INTO t29 VALUES (1), (2), (3);
+ALTER TABLE t29 ALTER COLUMN x TYPE INT4;
+
+query I
+SELECT x FROM t29 ORDER BY x
+----
+1
+2
+3
+
+# ColumnConversionValidate should error out if the conversion is not valid.
```

Comment:
This comment doesn't make much sense out of context -- can you give the context needed?

Reply:
Added context